src/bin/rgb.rs

```diff
@@ -382,11 +387,32 @@ fn main() -> Result<(), Error> {
             }
         },
         Command::Psbt { subcommand } => match subcommand {
-            PsbtCommand::Bundle { psbt_in, psbt_out } => {
+            PsbtCommand::Bundle {
+                psbt_in,
+                psbt_out,
+                method,
+            } => {
                 let psbt_bytes = fs::read(&psbt_in)?;
                 let mut psbt = Psbt::deserialize(&psbt_bytes)?;
 
+                if method == CloseMethod::TapretFirst {
+                    let output = psbt.outputs.last_mut().expect("PSBT should have outputs");
```

Comment:
Seems this is incorrect: why we are using the last output? `TapretFirst` implies use of the first taproot output, so outputs must be enumerated and looked for the first one which is a taproot output.

Reply:
now looking for the first taproot output